### PR TITLE
[Keymap] Adding HHKB Keymap for dz60rgb_wkl:v2

### PR DESCRIPTION
**Pull request by Macbet**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c (hunk 1 above)
Comment (Macbet) [reply keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c:1]: sure

Comment (Macbet) [reply keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c:17]: ready !

**Review by drashna — APPROVED** [review]

File: keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
+        CTL_T(KC_CAPS), KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSPO,        KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSPC, MO(1),
+        _______,     KC_LALT, KC_LGUI,                            KC_SPC,                                      KC_RGUI, KC_RALT, _______
+    ),
+    [1] = LAYOUT_60_tsangan_hhkb(
+        KC_PWR,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL, \
+        KC_TRNS, RGB_TOG, RGB_MOD, RGB_RMOD, BL_TOGG, BL_STEP, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS,   KC_UP, KC_TRNS, KC_TRNS,  \
+        KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT,       KC_PENT,       \
+        KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS,  KC_END, KC_PGDN, KC_DOWN,       KC_TRNS, KC_TRNS, \
```
Comment (spidey3) [inline keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c:31]: You don't need `\` at the end of these lines:
```suggestion
        KC_PWR,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL,
        KC_TRNS, RGB_TOG, RGB_MOD, RGB_RMOD, BL_TOGG, BL_STEP, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS,   KC_UP, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT,       KC_PENT,
        KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS,  KC_END, KC_PGDN, KC_DOWN,       KC_TRNS, KC_TRNS,
```

**Review by spidey3 — CHANGES_REQUESTED** [review]

File: keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c (hunk 2 above)
Comment (Macbet) [reply keyboards/dztech/dz60rgb_wkl/keymaps/hhkb/keymap.c]: fixed

**Review by spidey3 — APPROVED** [review]

**Pull request merged**